kuma/static/js/analytics.js

```diff
@@ -38,8 +38,7 @@
             };
 
             // If Analytics has loaded, go ahead with tracking
-            // Checking for ".create" due to Ghostery mocking of ga
-            if(ga && ga.create) {
```

Comment:
`ga.create` needs to be checked.  If the user has Ghostery, and we don't check for `ga.create`, the user just bricks when they click a link.